src/app/content/practiceQuestions/components/Answer/index.tsx

```diff
@@ -63,19 +72,22 @@ const Answer = ({
       id='i18n:practice-questions:popup:answers:choice'
       values={{choiceIndicator: choiceIndicator.toUpperCase()}}
     >{(msg: string) =>
-      <AnswerLabel aria-label={msg}>
-        {choiceIndicator}
+      <React.Fragment>
         <input
+          id={choiceIndicator}
```

Comment:
this input needs conditional `aria-selected` according to the spec right?

Reply:
I think it doesn't because screenreader (tested with ChromeVox) will read that you've checked the input radio button and when you TAB to this input you will be informed if it is or it is not selected